File: java/P0-Box-GettingStarted/src/rectangle/Box.java
package rectangle;

public class Box {
    
    private int length, width, height;
    
    public Box(int len, int wid) {
        length = len;
        width = wid;
        this.height = 0;
    }
    
    public Box(int len, int wid, int height) {
        length = len;
        width = wid;
        this.height = height;
    }
    
    public int perimeter() {
        int perim = (2 * length) + (2 * width);
        return perim;
    }
    
    public int area() {
        int area = length * width;
        return area;
    }
    
    public int volume() {
        int volume = this.length * this.width * this.height;
        return volume;
    }
    
}

File: c-sharp/P0-Box-GettingStarted/Rectangle/Box.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Rectangle
{
    public class Box
    {
        private int length, width;

        public Box(int len, int wid)
        {
            length = len;
            width = wid;
        }

        public int calcPerimeter()
        {
            int perim = (2 * length) + (2 * width);
            return perim;
        }

        public int calcArea()
        {
            int area = length * width;
            return area;
        }
    }
}
